cartography/intel/aws/sqs.py

```diff
@@ -0,0 +1,140 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_sqs_queue_list(boto3_session: boto3.session.Session, region: str) -> List[str]:
```

Comment:
Add `@aws_handle_regions` before this function. You'll also have to add `from cartography.util import aws_handle_regions` in the beginning.